tracer/src/Datadog.Trace/Debugger/Helpers/RuntimeHelper.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="RuntimeHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.Debugger.Helpers
+{
+    internal class RuntimeHelper
+    {
+        internal static bool IsNetOnward(int major)
+        {
+            return Environment.Version.Major >= major;
+        }
+
+        internal static bool IsCoreClr()
+        {
+#if NETCOREAPP
+            return true;
+#else
+            return false;
+#endif
+        }
```

Comment:
I think we already have this information available from `FrameworkDescription.Instance`. I would be wary about `IsNetOnward` to, with the complexity of .NET Core vs .NET FX vs .NET 5+

Reply:
Thanks! I've changed the code a bit, I'm basing the IsCoreClr check on `FrameworkDescription` in f2d4a493c8678ae1f255966ab96f4ce3c074568f.